From 584bb9e5668892d3a453cdae156527dff158dcb0 Mon Sep 17 00:00:00 2001
From: Hien Huynh <hien.huynh.px@renesas.com>
Date: Tue, 30 Jun 2020 11:14:06 +0700
Subject: [PATCH 039/458] arm64: dts: renesas: r8a774c0-ek874-idk-2121wr: Add
 support for dual channels LVDS

In order to support dual LVDS, we must set "dual-link" mode in lvds
nodes in devicetree.
This patch does this thing.

Signed-off-by: Hien Huynh <hien.huynh.px@renesas.com>
---
 arch/arm64/boot/dts/renesas/r8a774c0-ek874-idk-2121wr.dts | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/r8a774c0-ek874-idk-2121wr.dts b/arch/arm64/boot/dts/renesas/r8a774c0-ek874-idk-2121wr.dts
index 9aa7ce9..9d50061 100644
--- a/arch/arm64/boot/dts/renesas/r8a774c0-ek874-idk-2121wr.dts
+++ b/arch/arm64/boot/dts/renesas/r8a774c0-ek874-idk-2121wr.dts
@@ -93,6 +93,7 @@
 
 	clocks = <&cpg CPG_MOD 727>, <&x13_clk>;
 	clock-names = "fck", "dclkin.0";
+	mode = "dual-link";
 
 	ports {
 		port@1 {
@@ -108,6 +109,7 @@
 
 	clocks = <&cpg CPG_MOD 727>, <&x13_clk>;
 	clock-names = "fck", "dclkin.0";
+	mode = "dual-link";
 
 	ports {
 		port@1 {
-- 
2.7.4

